SEED-platform/seed
# Major mapping improvements

#### Any background context you want to provide?
#### What's this PR do?
- Logically reordered mapping columns
![image](https://user-images.githubusercontent.com/411466/41118367-f8728cf8-6a4c-11e8-9610-da725a8b4ba3.png)
- Restricted typeahead columns to only valid options on the currently selected property/taxlot distinction (`Jurisdiction Tax Lot ID` won't show up if you have `Property` selected on the current mapping row)
- Corrected display names, initial suggestions, typeahead suggestions, and translations of all fields during mapping (`UBID` instead of `Ubid`, `PM Property ID` instead of `Pm Property Id`)
- `extra_data` fields will no longer have auto-generated titlecased display names - they use the name exactly as it was typed
- Updated translations
- Removed deprecated code
- Disabled flippers (both are obsolete)

- Fixed typos
- Also fixed error message visibility when changing your password

#### How should this be manually tested?
Upload a file, try out the suggestions based on property/taxlot, try weird names, make sure that those weird names appear when reviewing the data and in the inventory view.
#### What are the relevant tickets?
#1172
